test/README.md

```diff
@@ -208,6 +219,48 @@ git clone user@gitserver:repos/test1.git
 make e2e-test
 ```
 
+### Running E2E with an existing cluster
+If you have a cluster already running, you can run the tests with:
+```bash
+export FLIGHTCTL_NS=flightctl
+export KUBEADMIN_PASS=your-oc-password-for-kubeadmin
+
+make in-cluster-e2e-test
+```
+
+#### If your host system is not suitable for bootc image builder
+
+* Install vagrant
+```bash
+sudo dnf install vagrant
+```
+* Install vagrant-libvirt plugin
+```bash
+vagrant plugin install vagrant-libvirt
+```
+
+* Continue inside vagrant
+
+```bash
+cp you-kubeconfig-file ~/.kube/config
+
+cd ~/flightctl
+
```

Comment:
I would add the step `sudo systemctl enable --now virtnetworkd`

Reply:
Added now, thanks :)